### DISCUSSION TITLE
Rendering rich text from a SharePoint document library multiline of text column in custom layouts

### DISCUSSION THREAD

**Discussion by jbmueller0584** [microsoft-search/pnp-modern-search#2129] [Q&A] [answered]
Hello dear community,

since SPO modern obviously now have implemented _multi lines of text as rich text in document libraries_ **finally** it would be great to render that rich text also in custom layouts.

Because when i specify rich content like so

![2022-06-02 11_25_04-2022-06-LA-2 pptx - PowerPoint](https://user-images.githubusercontent.com/94067337/171604153-f8e58b3b-15de-4797-b51f-16d895da286e.png)

and use that column in my custom layout

![2022-06-02 11_27_59-● studien-db-result-template-DE html - Visual Studio Code](https://user-images.githubusercontent.com/94067337/171604167-178696e4-ad02-4c5f-95cb-42abacf27348.png)

the content is printed out as plain text

![2022-06-02 11_30_52-Suchergebnisse und 94 weitere Seiten - JBM – Microsoft​ Edge](https://user-images.githubusercontent.com/94067337/171604184-fcfe5a7a-653d-4e74-9856-f9b684eebd09.png)

The reason behind is, because its rendered like this

![2022-06-02 13_34_55-Suchergebnisse und 97 weitere Seiten - JBM – Microsoft​ Edge](https://user-images.githubusercontent.com/94067337/171620786-353a31d3-540e-43e6-8973-1d93364d6c63.png)

Is there a function in handlebars like encode/decode HTML?

Am i doing something wrong? Any new information or an update of the modern search is highly appreciated!

Thanks a lot for everything you guys do 🙏👍👏

Cheers,
Joerg

**Comment by wobba** [comment, marked as answer]
See #533

**Comment by jbmueller0584** [comment]
Amazing !!!!!!!!!! Thank you !!!
